api/v1alpha1/provisioning_types.go

```diff
@@ -110,6 +120,9 @@ type ProvisioningSpec struct {
 	// openshift-machine-api namespace. When set to true, this provisioning
 	// configuration would be used for baremetal hosts across all namespaces.
 	WatchAllNamespaces bool `json:"watchAllNamespaces,omitempty"`
+
+	// BootIsoSource
```

Comment:
This should contain docs about the field. It's user-facing and shows up in the `explain` output for a CRD.